frontend/src/metabase/admin/settings/components/widgets/SettingInput/SettingInput.tsx

```diff
@@ -45,6 +54,7 @@ const SettingInput = ({
         SettingsInput: type !== "password",
         SettingsPassword: type === "password",
       })}
+      normalize={normalizeWithProps}
```

Comment:
The reason we need to also pass `normalize` to the inner component is because it's possible for `SettingInputBlurChange` to have different value than this component. e.g. if we have `/` and we start adding more spaces `SettingInputBlurChange` would have a state of `/              ` while the normalize function would strip it down to `/` again, this means the`value` prop passed to it would be the same, and it won't know when it should copy the prop to its `internalState`.
https://github.com/metabase/metabase/blob/6b1fab0f3b2a31e84743d99e47a314f9d0463eb2/frontend/src/metabase/components/InputBlurChange/InputBlurChange.tsx#L31

Passing `normalize` ensures we always normalize the `internalState` as well so that it would stay in sync with `setting.value`